gdcmodels/__init__.py

```diff
@@ -35,7 +49,10 @@ def get_es_models():
                 es_models[es_index][es_type] = {
                         '_mapping': load_yaml(pj(es_model_dir, es_index, f))
                     }
-
+                defs_name = pj(es_model_dir, es_index,es_type+".".join(['_definitions', 'yaml']))
```

Comment:
```suggestion
                defs_name = pj(es_model_dir, es_index, es_type + '_definitions.yaml')
```
don't really see the need of the `join` here

Reply:
simpler, I like it